### PR TITLE
Create 课程.md

### PR DESCRIPTION
**Pull request by SafaFIMOUD**
Translate to Chinese

### CODE REVIEW

**Comment by barraponto** [comment]
Hey @SafaFIMOUD , I don't really speak chinese, so it's hard for me to review.
Can you get someone else to review it?

**Comment by SafaFIMOUD** [comment]
hello @barraponto , no acctually i cant get someone else to review it, do you want me to do another language or another exercice (annotations, quizzes...) ?

**Comment by barraponto** [comment]
Ok. Can you move the chinese translation to another branch? I don't want to lose it, I just don't want to merge it in this PR

**Comment by SafaFIMOUD** [comment]
Done.

Here is the new branch with chinese translation : https://github.com/barraponto/usegit/pull/23

File: quizzes/Quiz-Basics.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+These 5 Git quiz questions will test your knowledge on a variety of topics, including the use of basic Git commands. Good luck. These Git quiz questions aren't easy.
+
+## Question 1
+
+1. What is the default text editor for the Bash shell with a Windows-based Git install?
+
+   - [ ] Emacs
+   - [ ] Vim
+   - [ ] Notepad++
+   - [ ] Bash
```
Comment (barraponto) [inline quizzes/Quiz-Basics.md:12]: What is? I don't know!
You can make a copy of this file, call it Quiz-Basics-rubric and answer it there!

**Pull request merged**